tests/kernels/test_attention.py

```diff
@@ -181,7 +198,27 @@ def test_paged_attention(
 
     # Call the paged attention kernel.
     output = torch.empty_like(query)
-    if version == "v1":
+    if device == "hpu":
+        print(key_cache.shape, value_cache.shape)
+        key_cache_hpu = key_cache.permute((0, 3, 1, 2, 4)).flatten(3)
+        value_cache_hpu = value_cache.permute((0, 3, 1, 2))
```

Comment:
I see only two ways to fix kvcache shape in this test: do these permute operations after each kv_cache_factory or change kv_cache_factory. 
I decided that for the test, there is no performance limitation, so it can be done in a less aggressive way(by changing only test hpu branch)